algo/src/main/java/com/hartwig/actin/algo/evaluation/general/HasMaximumBMI.java

```diff
@@ -0,0 +1,69 @@
+package com.hartwig.actin.algo.evaluation.general;
+
+import com.hartwig.actin.PatientRecord;
+import com.hartwig.actin.algo.datamodel.Evaluation;
+import com.hartwig.actin.algo.datamodel.EvaluationResult;
+import com.hartwig.actin.algo.datamodel.ImmutableEvaluation;
+import com.hartwig.actin.algo.evaluation.EvaluationFactory;
+import com.hartwig.actin.algo.evaluation.EvaluationFunction;
+import com.hartwig.actin.clinical.datamodel.BodyWeight;
+import com.hartwig.actin.clinical.sort.BodyWeightDescendingDateComparator;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Optional;
+
+public class HasMaximumBMI implements EvaluationFunction {
```

Comment:
In general the functions have a name referring to "What needs to be true for this function to pass?". 

The current name may suggest the patient needs to have a certain maximum BMI. Maybe a name like "HasLimitedBMI" is more correct?

Reply:
I can see that my naming is not entirely consistent either! 

I like your latest suggestion: `HasBMIUpToLimit`

Reply:
Renamed in #4.